UI/src/view/createdLinearAssetsWorkList.js

```diff
@@ -0,0 +1,122 @@
+(function (root) {
+  root.CreatedLinearAssetWorkList = function() {
+    WorkListView.call(this);
+    var me = this;
+    this.hrefDir = "#work-list/createdLinearAssets";
+    this.title = 'Lisätyt viivamaiset kohteet';
+    var backend;
+    var showFormBtnVisible = true;
+    var assetConfig = new AssetTypeConfiguration();
+
+    var availableAssetsTypeId = [190];
+
+    this.initialize = function(mapBackend) {
+      backend = mapBackend;
+      me.bindEvents();
+    };
+
+    this.bindEvents = function () {
+      eventbus.on('createdLinearAssets:select', function() {
+        $('.container').hide();
+        $('#work-list').show();
+        me.generateWorkList(availableAssetsTypeId);
+      });
+    };
+
+    this.createVerificationForm = function(assetType) {
+      $('#tableData').hide();
+      $('.filter-box').hide();
+      if (showFormBtnVisible) $('#work-list-header').append($('<a class="header-link"></a>').attr('href', me.hrefDir).html('Kuntavalinta').click(function(){
```

Comment:
After the merge with master, have a look on this to see if the table fits. There were some CSS related corrections done to work lists on master recently.